### ISSUE TITLE
Think of some interesting trigger mechanism that would ban *everyone* in the server

### ISSUE THREAD

**Issue by gorel** [gorel/discord_dicebot#70]
I'm thinking we could have a concept of a "server-wide" ban where *everyone* gets shamed. Trigger could be if we get $reaction_threshold :BAN: reactions against the bot.

This gif should be sent whenever that happens:

https://media.tenor.com/OFvu8fsmumwAAAAd/oprah-winfrey-everybody-get-banned.gif

**Comment by glandsly** [comment]
I can't think of a funny trigger. Maybe 69 or 420 related or some prompt that justifies a "yall need jesus" response from Leeroy.

I like the idea of using this gif - https://giphy.com/gifs/one-direction-zayn-malik-abhhhhhhhhhhhhhhhhhhhhhhhh-P7JmDW7IkB7TW

**Comment by gorel** [comment]
Maybe if there are two 1's rolled in a row?

**Comment by glandsly** [comment]
Maybe multiple roll violations in a day? Like rolling too often or trying to gamble over the limit.

**Comment by gorel** [comment]
One idea I had was two 1's in a roll = server ban for 24 hours

**Comment by glandsly** [comment]
Yeah I think that would be fine, was just trying to think of other triggers.

**Comment by jrodal98** [comment]
Leeroy gets 5 ban reaccs -> everyone gets banned except for Leeroy.

Something something "Collective punishment"
